src/sssom/util.py

```diff
@@ -1036,6 +1035,11 @@ def to_mapping_set_dataframe(doc: MappingSetDocument) -> MappingSetDataFrame:
     return MappingSetDataFrame.from_mapping_set_document(doc)
 
 
+DICT_FROM_MAPPING_ENUM_KEYS = set(_get_sssom_schema_object().dict["enums"].keys())
+SLOTS = _get_sssom_schema_object().dict["slots"]
+DOUBLE_SLOTS = {k for k, v in SLOTS.items() if v["range"] == "double"}
```

Comment:
Add the three variables above as member vars in the schemaview object? 

And then use

sssom_schema = _get_sssom_schema_object()

and the use in code:

sssom_schema.slots
sssom_schema.double_slots
sssom_schema.mapping_enum_keys (the word `dict` is confusing since the datatype is `set`).




sc

Reply:
Done